system-test/header.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as assert from 'assert';
+import {WebRiskServiceV1Beta1Client} from '../src/v1beta1';
+import * as protoTypes from '../protos/protos';
+
+const http2spy = require('http2spy');
+describe('header', () => {
+  it('populates x-goog-api-client header', async () => {
+    const client = new WebRiskServiceV1Beta1Client();
+    const request = {
+      uri: 'http://testsafebrowsing.appspot.com/s/malware.html',
+      threatTypes: ['MALWARE'],
+    };
+    await client.searchUris(request as unknown as protoTypes.google.cloud.webrisk.v1beta1.SearchUrisRequest);
```

Comment:
by the way, why do we cast to `unknown` here?  is the type wrong?

Reply:
yea, the two types are not compatible, so the local request lacks `toJSON` method which is not assignable to `google.cloud.webrisk.v1beta1.SearchUriRequest`[toJSON method is required, not optional]